botorch/posteriors/torch.py

```diff
@@ -78,6 +78,14 @@ def __getattr__(self, name: str) -> Any:
         """
         return getattr(self.distribution, name)
 
+    def __getstate__(self) -> Dict[str, Any]:
+        r"""A minimal utility to support pickle protocol."""
```

Comment:
My be worth explaining in a comment in code why we need this (b/c of the definition of `__getattr__` on this class)